### ISSUE TITLE
Accessibility - Projects - Issues with search form

### ISSUE THREAD

**Issue by terrimorton** [dotnet-foundation/website#309]
Page: https://dotnetfoundation.org/projects
Environment: Chrome version 84.0.4147.125

There are 3 accessibility issues on this page.

![2020-08-17_15-36-09](https://user-images.githubusercontent.com/4643903/90436986-94daa380-e09f-11ea-9d82-9759300ae48e.jpg)

1. Ensures every id attribute value of active elements is unique
Title: WCAG 4.1.1: Ensures every id attribute value of active elements is unique (input[value="project"])
Tags: Accessibility, WCAG 4.1.1, duplicate-id-active
Issue: Ensures every id attribute value of active elements is unique (duplicate-id-active - https://dequeuniversity.com/rules/axe/3.5/duplicate-id-active?application=msftAI)
Target application: .NET Foundation Projects - https://dotnetfoundation.org/projects
Element path: input[value="project"]
Snippet: `<input id="type" name="type" type="radio" value="project" checked="">`
How to fix: 
Fix any of the following:

-   Document has active elements with the same id attribute: type

2. Ensures every form element has a label
Title: WCAG 1.3.1,WCAG 3.3.2: Ensures every form element has a label (#textToSearch)
Tags: Accessibility, WCAG 1.3.1, WCAG 3.3.2, label
Issue: Ensures every form element has a label (label - https://dequeuniversity.com/rules/axe/3.5/label?application=msftAI)
Target application: .NET Foundation Projects - https://dotnetfoundation.org/projects
Element path: #textToSearch
Snippet: `<input class="form-control" id="textToSearch" name="textToSearch" type="search" value="" placeholder="">`
How to fix: Fix any of the following:
-   aria-label attribute does not exist or is empty
-   aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
-   Form element does not have an implicit (wrapped) <label>
-   Form element does not have an explicit <label>
-   Element has no title attribute or the title attribute is empty

3. Ensures buttons have discernible text
Title: WCAG 4.1.2: Ensures buttons have discernible text (#search-projects)
Tags: Accessibility, WCAG 4.1.2, button-name
Issue: Ensures buttons have discernible text (button-name - https://dequeuniversity.com/rules/axe/3.5/button-name?application=msftAI)
Target application: .NET Foundation Projects - https://dotnetfoundation.org/projects
Element path: #search-projects
Snippet: `<button id="search-projects" class="btn btn-primary mb-2"><i class="fa fa-search"></i></button>`
How to fix: Fix any of the following:

-   Element does not have inner text that is visible to screen readers
-   aria-label attribute does not exist or is empty
-   aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
-   Element's default semantics were not overridden with role="presentation"
-   Element's default semantics were not overridden with role="none"
-   Element has no title attribute or the title attribute is empty

**Comment by terrimorton** [comment]
Closing this ticket and opening 3 separate tickets which may be easier to knock off 1-by-1.

#862 #863 #864
